### DISCUSSION TITLE
Is it possible to load a book into the reader from the terminal emulator?

### DISCUSSION THREAD

**Discussion by jayjf** [koreader/koreader#9543] [Q&A]
Is it possible to load a book into the reader from the terminal emulator?
In other words, if I have the path to a book file, can a give some command, with the book file as an argument, and the book will load just as if I had tapped it in History or from the File Browser?

**Comment by NiLuJe** [comment]
Not that I know of, no.

**Reply by Frenzie** [reply]
I don't think we have something like that right now but you can drag and drop.

**Reply by NiLuJe** [reply]
Oh, wait, I was thinking of the terminal emulator *plugin*, not on Desktop ;o).

But, yeah, the point still stands ;).

**Reply by Frenzie** [reply]
Hm, now I'm thinking you were right and I was wrong. xD
